### PR TITLE
fields formating in targets

### PR DESCRIPTION
**Pull request by btry**

### CODE REVIEW

File: inc/field.class.php [hunk 1]
```diff
@@ -25,9 +29,22 @@ public function __construct($fields, $data = []) {
...
+      return $input;
+   }
+
+   /**
+    * Prepares a answer value for output in a target object
+    * @param string|array $input the answer to format for a target (ticket or change)
+    * @return string
+    */
+   public function prepareQuestionInputForTarget($input) {
+      return addslashes($input);
```
Comment (orthagh) [inline inc/field.class.php]: Toolbox::addslashes_deep maybe ?

Comment (btry) [reply inc/field.class.php]: You're right. This is safer

Comment (orthagh) [reply inc/field.class.php]: apart this, it should be ok

Comment (btry) [reply inc/field.class.php]: thanks !

**Pull request merged**